package.json

```diff
@@ -9,9 +9,11 @@
   "repository": "stylelint/remark-preset",
   "license": "MIT",
   "author": "stylelint",
-  "main": "index.js",
+  "type": "module",
+  "exports": "./index.js",
```

Comment:
[note] `main` is replaced with `exports` via 1ebb623. See the guide ["Pure ESM package"](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c).